docs/tests/auth-github.md

```diff
@@ -0,0 +1,54 @@
+# GitHub Auth Benchmark `github_auth`
+
+This benchmark will test GitHub Authentication to Vault. The primary required fields are `organization` and `token`.
+
+## Test Parameters
+
+### Auth Configuration `auth`
+
+- `organization` `(string: <required>)` - The organization users must be part of.
+- `organization_id` `(int: 0)` - The ID of the organization users must be part of. Vault will attempt to fetch and set this value if it is not provided.
+- `base_url` `(string: "")` - The API endpoint to use. Useful if you are running GitHub Enterprise or an API-compatible authentication server.
+- `token_ttl` `(integer: 0 or string: "")` - The incremental lifetime for generated tokens. This current value of this will be referenced at renewal time.
```

Comment:
```suggestion
- `token_ttl` `(string: "")` - The incremental lifetime for generated tokens. This current value of this will be referenced at renewal time.
```